doc/development_process/dev_env_and_tools.rst

```diff
@@ -294,7 +294,8 @@ Issue priority labels
 =============  ===============================================================
 Labels         ``priority:{high|medium|low}``
 Applicable to  Issues only
-Description    To classify the impact and importance of a bug or feature
+Description    To classify the impact and importance of a bug or
+               `feature <feature-tracking>`
```

Comment:
This depends on #15340 being merged first (that's why you're getting the doc build error). 

For clarity, it would be better to write these references to other documents as   ``:ref:`feature <feature-tracking>` `` though recent changes to our Sphinx configuration will now hunt for the right domain to reference this ``feature-tracking`` id.
``